### PR TITLE
Adding french version of the man page

### PR DESCRIPTION
**Pull request by glc-mvion**
Resolving #1 (:

### CODE REVIEW

File: pkg/colorful-french.man [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.\" Manpage for colorful
+.\" Contact veg@svgames.pl to correct errors or typos.
+.TH man 6 "2014-03-16" "1.2" "Games Manual"
```
Comment (suve) [inline pkg/colorful-french.man]: "Games Manual" should also be translated.

File: pkg/colorful-french.man [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+.TP
+\fB\-\fR/\fB+\fR
+Augmenter / diminuer le volume
+.TP
+\fBP\fR
+Mettre en pause / Reprendre le jeu
+.TP
+Toutes ces touches peuvent être redéfinies.
+.SH DOSSIERS
+Les données du jeu se trouvent dans : \fB/usr/share/suve/colorful\fR.
```
Comment (suve) [inline pkg/colorful-french.man:29]: Please remove the spaces before the colons.

File: pkg/colorful-french.man (hunk 1 above)
Comment (suve) [reply pkg/colorful-french.man]: I checked with `man-pages(7)` and the date should reflect the time the manpage received substantial changes. A translation is rather substantial, so please use a current date. :)

**Review by suve — CHANGES_REQUESTED** [review]
Just a few minor tweaks and it'll be good to go.

File: pkg/colorful-french.man (hunk 1 above)
Comment (glc-mvion) [reply pkg/colorful-french.man]: Fixed those remarks !

File: pkg/colorful-french.man (hunk 2 above)
Comment (glc-mvion) [reply pkg/colorful-french.man:29]: But in french typographics rules, colons must be surrounded by space, does it need special escaping ?

Comment (suve) [reply pkg/colorful-french.man:29]: Oh, sorry, did not realise that. Let's leave it like this, then.

**Review by suve — APPROVED** [review]

**Comment by suve** [comment]
Everything looks fine now; merged. Thanks!

**Comment by glc-mvion** [comment]
You're welcome ! Happy Hacktoberfest !

**Pull request merged**